src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,39 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
```

Comment:
How about just remove the error condition by taking care of it inside of `getValidationTime`:  https://github.com/HowardHinnant/rippled/commit/1fd1124289778cc55b60f85b89e1d04a3ae93cea
?